src/platform/datapath_winkernel.c

```diff
@@ -1293,7 +1293,8 @@ SocketCreateUdp(
         Status = QUIC_STATUS_OUT_OF_MEMORY;
         goto Error;
     }
-
+    
+    Binding->ReserveAuxTcpSock = FALSE;
```

Comment:
Binding is zero'ed a few lines below.

Reply:
It does, but we set *NewBinding before we zero it out so I am wondering perhaps if there is a race condition.